collections/ansible_collections/demo/satellite/roles/register_host/vars/RedHat8.yml

```diff
@@ -3,3 +3,6 @@ rhsm_enabled_repos:
   - rhel-8-for-x86_64-baseos-rpms
   - rhel-8-for-x86_64-appstream-rpms
   - satellite-client-6-for-rhel-8-x86_64-rpms
+instance_name: "{{ inventory_hostname | regex_replace('_','-') }}"
+rex_user: root
+# rex_user: "{{ ansible_user }}"
```

Comment:
+1 for rex_user defaulting to ansible_user. That would be a service account in the environment that is already used for ssh management. Users could override with `root` if they wanted to take a less secure approach.

Reply:
I agree. Will move to defaults/main.yml